test/absinthe/middleware/dataloader_test.exs

```diff
@@ -158,11 +159,57 @@ defmodule Absinthe.Middleware.DataloaderTest do
       ]
     }
 
+    # Get test PID and clear mailbox
+    self = self()
+    flush_process_mailbox()
```

Comment:
Why do you need to do this?

Reply:
Fixed root cause of test flakiness by making tests non-async. Deleted this block